platform/saivpp/vpplib/SaiObjectDB.h

```diff
@@ -45,8 +53,14 @@ namespace saivpp
 
         virtual sai_status_t get_attr(_Out_ sai_attribute_t &attr) const = 0;
 
-        // Get the manditory attribute of the object. If not found, log an error and return SAI_STATUS_FAILURE
-        virtual sai_status_t get_mandatory_attr(_Out_ sai_attribute_t &attr) const = 0;
+        /**
+         * @brief Retrieves the value of a specific attribute of the SAI object. If the attribute is not found, log an error and 
+         *        return SAI_STATUS_FAILURE.
+         * 
+         * @param attr [out] A reference to the sai_attribute_t structure to store the attribute value.
+         * @return sai_status_t The status of the attribute retrieval operation.
+         */
+        sai_status_t get_mandatory_attr(_Out_ sai_attribute_t &attr) const;
```

Comment:
Not sure about the exact convention, but shouldn't attr be `_Inout_` in `get_<mandatory>_attr`?